src/lib/PostgrestTransformBuilder.ts

```diff
@@ -120,4 +120,51 @@ export default class PostgrestTransformBuilder<T> extends PostgrestBuilder<T> {
     this.headers['Accept'] = 'text/csv'
     return this as PromiseLike<PostgrestSingleResponse<string>>
   }
+
+  /**
+   * Set the response type to GeoJSON.
+   */
+  geojson(): PromiseLike<PostgrestSingleResponse<string>> {
```

Comment:
Can we have the proper type here? Doesn't look like `data` is `string` looking at the test